RetailCoder.VBE/UI/Settings/IndenterSettingsViewModel.cs

```diff
@@ -262,14 +263,72 @@ public int IndentSpaces
         }
 
         // ReSharper disable once InconsistentNaming
-        private const string _previewSampleCode = @"Sub Foo()
-    ' Do something here
-    ' xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx ' comment here
-End Sub";
+        private const string _previewSampleCode =
+@"' Example Procedure
+Sub ExampleProc()
+
+    ' SMART INDENTER
+    '@ 2016 by Rubberduck VBA.
+
+    Dim iCount As Integer
+    Static sName As String
+
+    If YouWantMoreExamplesAndTools Then
+        ' Visit http://rubberduckvba.com/
```

Comment:
If YouLikeRubberduck Then
        ' Star us on GitHub http://github.com/rubberduck-vba/Rubberduck
        ' Follow us on Twitter @rubberduck-vba/contributors 
        ' Visit http://rubberduckvba.com for news and updates